FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Nicolò Pasini on 10/09/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+final class ManagedFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var cache: ManagedCache
+}
+
+extension ManagedFeedImage {
+	static func images(from localFeed: [LocalFeedImage], in context: NSManagedObjectContext) -> NSOrderedSet {
+		NSOrderedSet(array: localFeed.map({ local in
```

Comment:
Suggestion:
Using implicit `return` is absolutely ok. On the other hand implicit `return`s are not used in this project thus it is better to be consistent and follow the project's code style
```suggestion
		return NSOrderedSet(array: localFeed.map({ local in
```